airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/legacy_typing_deduping/AlterTableReport.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.legacy_typing_deduping
+
```

Comment:
copy of the old AlterTableReport, but I dropped the `isDestinationV2Format`, and simplified the implementation.